dewy-client/README.md

```diff
@@ -0,0 +1,124 @@
+# dewy-client
+A client library for accessing Dewy Knowledge Base API
+
+## Usage
+First, create a client:
+
+```python
+from dewy_client import Client
+
+client = Client(base_url="https://api.example.com")
```

Comment:
Might be good to show a more complete example here - loading a doc and querying for example.

Reply:
My plan is to update this once I've used the client in a demo (planning to work on a chainlit app). Currently, I'm not 100% sure on these since we don't have a published API, etc.